build.sbt

```diff
@@ -17,7 +17,8 @@
 
 import scala.collection.immutable.ListSet
 
-import sbtcc._
+import Dependencies.genjavadoc
+import sbtcc.*
```

Comment:
Is this import * correct, shouldn't it be _ in scala?

Reply:
Tools are suggesting * now. I think scala 3 preferred style is * not _ for several things _ was being used for. Scala 2 did have far too many overloads of things on the _ character.